### ISSUE TITLE
start_outputs和end_outputs维度问题

### ISSUE THREAD

**Issue by lxgend** [JavaStudenttwo/BERT_MRC#1]
https://github.com/JavaStudenttwo/BERT_MRC/blob/69b416471004b4c9ad352c2bf94d9f3ff2d3766b/model/bert_mrc.py#L26

请教一下，start_outputs和end_outputs的nn.Linear(config.hidden_size, 2) 中的2，代表什么意思？
我看香农官方的代码，他们又改成了nn.Linear(config.hidden_size, 1)
非常迷惑，烦请解答，谢谢

**Comment by Senwang98** [comment]
@lxgend 
不知道你后续理解了没，我最近在看ner的工作，这个维度2是ce loss，维度1是为了bce。

顺便问一下兄弟，官方的代码你复现精度了吗？我感觉跟paper中的结果有差距
